stdlib/test/builtin/test_int.mojo

```diff
@@ -19,6 +19,8 @@ from testing import assert_equal, assert_true, assert_false, assert_raises
 from python import PythonObject
 from memory import UnsafePointer
 
+alias Bytes = List[Byte]
```

Comment:
Suggestion: This definition is not decided over yet. could you take this into the scope of your test?

Reply:
Sure! Done: https://github.com/msaelices/mojo/commit/7766ea3487b6859d13d92f4e73507f3f65d5c83d